### DISCUSSION TITLE
Input Thread Stuttering

### DISCUSSION THREAD

**Discussion by Neppsz** [ppy/osu#30562] [Q&A]
### Type

Performance

### Bug description

I was playing the game fine a few days ago, but randomly a day or two ago I started to get these weird mouse stutters where my cursor would freeze then teleport to where I moved it to. The performance graph shows only the input thread receiving this lag.

### Screenshots or videos

https://github.com/user-attachments/assets/5ef115cc-0935-47d7-b6de-12c7b10d1d82

### Version

2024.1009.1-lazer

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/17686166/compressed-logs.zip)

**Comment by smoogipoo** [comment]
In general, this is going to be caused by some background program.
